pkg/k8shandler/elasticsearch_helper.go

```diff
@@ -158,7 +158,7 @@ func updateAllIndexTemplateReplicas(clusterName, namespace string, client client
 
 						templateJson, _ := json.Marshal(template)
 
-						logrus.Debugf("Updating template %v from %d replicas to %d", templateName, currentReplicas, replicaCount)
+						logrus.Debugf("Updating template %v from %v replicas to %d", templateName, currentReplicas, replicaCount)
```

Comment:
shouldn't this still be `%d` since its a number?

Reply:
it failed to run tests with `%d`

Reply:
`currentReplicas, ok := index["number_of_replicas"].(string)` it does not seem like a string.

Reply:
that's odd... in that case `ok` should be `false` and we shouldn't even get to this line...

Reply:
that does not prevent the compiler from complaining. You cannot change the type once it is defined as string.